### PR TITLE
Fix compatibility with latest Android Auto

### PR DESCRIPTION
**Pull request by abraha2d**
Credit goes to @borconi for the updated proto files.

fixes opencardev/crankshaft#357.

### CODE REVIEW

File: aasdk_proto/WifiChannelData.proto [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+*  aasdk is distributed in the hope that it will be useful,
+*  but WITHOUT ANY WARRANTY; without even the implied warranty of
+*  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+*  GNU General Public License for more details.
+*
+*  You should have received a copy of the GNU General Public License
+*  along with aasdk. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+syntax="proto3";
```
Comment (gregorhollmig) [inline aasdk_proto/WifiChannelData.proto]: Why is everything proto2 beside this one message?

Comment (borconi) [reply aasdk_proto/WifiChannelData.proto]: Sorry that should be proto2 as well.

Comment (abraha2d) [reply aasdk_proto/WifiChannelData.proto]:
```suggestion
syntax="proto2";
```

**Comment by abraha2d** [comment]
That latest change to WifiChannelData.proto hasn't been tested. I can test it when I get home later today, or if someone else can get to it before me.

**Comment by gregorhollmig** [comment]
Who has write access and can merge this PR?

**Comment by htruong** [comment]
I can merge. I wonder if anyone can help me validate and merge, but wondering if anyone can help us with maintaining.

**Comment by abraha2d** [comment]
@htruong I can volunteer to help maintain this.

**Comment by htruong** [comment]
Thanks @abraha2d - What's your email address?

**Comment by abraha2d** [comment]
@htruong I sent you an email at htruong@tnhh.net.

**Pull request merged**